### DISCUSSION TITLE
Is any chance for just replace linux kernel of older buildroot Pi 4 image for Pi5

### DISCUSSION THREAD

**Discussion by Vichezo-72** [community/community#130096] [Programming Help]
### Body

Hi every one, I was working on the buildroot image building. Originally the raspi 1bp is used in our product, but we upgrade the PI to raspi3bp. So the software includes Qt and Python need to upgrade to new version for new version buildroot. But now the raspi5 is coming, I hope to change the hardware from raspi3bp to raspi 5. But looks need using the latest version buildroot and also the python and Qt version are upgraded not curretly I used. With new version Qt and python, some module and package not supported. All my software even the open source I used looks need much modification to working with new version Qt and Python. My question is any chance to easier change the rpi-firmware and start up files only, not necessary to build whole new buildroot image with new Linux Kernel. Could any suggestion for me. thank you in advance.

### Guidelines

- [X] I have read and understood this category's [guidelines](https://github.com/orgs/community/discussions/51384) before making this post.

**Comment by HemalathaT2** [comment]
To send a private message when a client sends the foo command, you can use the discord.py library
import discord

client = discord.Client()

@client.event
async def on_message(message):
    if message.content.startswith('foo'):
        user = message.author
        await user.send('This is a private message!')
        await message.channel.send('Private message sent!')

client.run('YOUR_DISCORD_BOT_TOKEN')
